src/main/kotlin/app/revanced/patches/grindr/unlimited/fingerprints/HasFeatureFingerprint.kt

```diff
@@ -0,0 +1,17 @@
+package app.revanced.patches.grindr.unlimited.fingerprints
+
+import app.revanced.patcher.extensions.or
+import app.revanced.patcher.fingerprint.method.annotation.FuzzyPatternScanMethod
+import app.revanced.patcher.fingerprint.method.impl.MethodFingerprint
+import com.android.tools.smali.dexlib2.AccessFlags
+import com.android.tools.smali.dexlib2.Opcode
+
+//a
+
+object HasFeatureFingerprint : MethodFingerprint(
+    "Z",
+    parameters = listOf("Lcom/grindrapp/android/model/Feature;"),
+    customFingerprint = { methodDef, _ ->
+        methodDef.name.contains("a")
```

Comment:
This will not work across versions, you should not directly check for obfuscated names.